<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EVgui</class>
 <widget class="QMainWindow" name="EVgui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EVgui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="pic1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>处理画面</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="text">
     <string>秸秆占比：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ratio">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="cursorPosition">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>110</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>原图</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pic</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>EVgui</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pic1</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>EVgui</receiver>
   <slot>process()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>flag2()</slot>
  <slot>txt()</slot>
  <slot>start()</slot>
  <slot>process()</slot>
 </slots>
</ui>
